src/packageSourceGenerator/PackageSourceGeneratorTask/AddSbrpAttribute.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.SourceBuild.Tasks
+{
+    public partial class AddSbrpAttribute : Task
```

Comment:
Naming question.  What do you envision as the scope of this task?  If we needed to add additional tweaks to the source, would this task be expanded or would these be another task added?  From a performance aspect, the file IO would be expensive to utilize multiple tasks.  With this in mind, what do you think about the name of this task?

Reply:
If we need to add additional attributes, I can see how running this task multiple times would be expensive. I'll change the name and adjust the task to have a parameter that is a list of attributes to add. This should eliminate needing to read and write the file multiple times.

Reply:
Refactored in this commit to add arbitrary attributes: https://github.com/dotnet/source-build-reference-packages/pull/794/commits/b6e2ed619dbe94e19bd75d42a170a22809445027. This allows the scope of the task to be wider and eliminates writing to the file more than once.